src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:first-child {
+  margin: 0;
+}
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
Code duplication. Enumerate the selectors separated by commas and add a background to them